### PR TITLE
Add option to enforce cipher order

### PR DESCRIPTION
**Pull request by acdemyers**
Adding an option to enforce cipher order on servers.

### CODE REVIEW

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

File: dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -51,6 +51,8 @@ repertoire.verifyHostname=Host name verification
 repertoire.verifyHostname.desc=Specifies whether host name verification for outbound connections using a specific SSL configuration is enabled. If set to true, then all outbound SSL connections that use the specified SSL configuration undergo verification of the target server host name against that server's certificate. The attribute is set to false by default.
 repertoire.trustDefaultCerts=Trust the default certificates
 repertoire.trustDefaultCerts.desc=Specifies whether the trust manager can establish trust by using the default certificates. If set to true, then the default certificates are used in addition to the configured truststore file to establish trust. The attribute is set to false by default.
+repertoire.enforceCipherOrder=Enforce cipher order on server side
+repertoire.enforceCipherOrder.desc=Specified whether to indicate on the server socket to enforce the cipher order.  If set to true, the server socket will be enabled to enforce cipher order.  The attribute is set to false by default.
```
Comment (ManasiGandhi) [inline dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties:55]: Change,
"Specified whether to indicate on the server socket to enforce the cipher order.  If set to true, the server socket will be enabled to enforce cipher order.  The attribute is set to false by default."
to,
"Specify on the server socket whether to enforce the cipher order. If set to true, the server socket is enabled to enforce the cipher order. The attribute is set to false by default."

**Review by ManasiGandhi — COMMENTED** [review]
Some edit suggestions.

File: dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (acdemyers) [reply dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties]: Thank you for reviewing.  I've made the suggested update.

Comment (ManasiGandhi) [reply dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties]: Thanks. Looks good.

File: dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -51,6 +51,8 @@ repertoire.verifyHostname=Host name verification
 repertoire.verifyHostname.desc=Specifies whether host name verification for outbound connections using a specific SSL configuration is enabled. If set to true, then all outbound SSL connections that use the specified SSL configuration undergo verification of the target server host name against that server's certificate. The attribute is set to false by default.
 repertoire.trustDefaultCerts=Trust the default certificates
 repertoire.trustDefaultCerts.desc=Specifies whether the trust manager can establish trust by using the default certificates. If set to true, then the default certificates are used in addition to the configured truststore file to establish trust. The attribute is set to false by default.
+repertoire.enforceCipherOrder=Enforce cipher order on server side
```
[the remark this answers is not in the document]
Comment (acdemyers) [reply dev/com.ibm.ws.ssl/resources/OSGI-INF/l10n/metatype.properties]: Thanks for the review.  The message as been update.

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#run-libby-bot

**Comment by skasund** [comment]
L2 message review not required

**Comment by acdemyers** [comment]
#build

**Comment by acdemyers** [comment]
#run-libby-bot

**Comment by acdemyers** [comment]
#build

**Review by emilytee — APPROVED** [review]

**Pull request merged**